### PR TITLE
Unified test documentation: UEFI shell and UEFI compatible interface

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemysław Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/30M-uefi-compatible-interface.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Dasharo Compatibility: UEFI compatible interface
+
+## Test cases
+
+### EFI001.001 Boot into UEFI OS (Ubuntu 20.04)
+
+**Test description**
+
+This test verifies the presence of UEFI compatible interfaces by booting
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/dasharo-compatibility/30M-uefi-compatible-interface.md:9]: Improved

File: docs/unified-test-documentation/dasharo-compatibility/30M-uefi-compatible-interface.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+1. Select the USB stick and press `Enter`.
+1. Wait for the `OPERATING_SYSTEM` to boot finalize booting, by either of the:
+   - `OPERATING_SYSTEM` installer initialization,
+   - login form initialization.
+1. Power OFF the DUT.
+
+**Expected result**
+
+1. Either the login screen or the `OPERATING_SYSTEM` installer appears on the
+   internal LCD.
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/dasharo-compatibility/30M-uefi-compatible-interface.md:74]: New line has already been added.

File: docs/unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+1. `FIRMWARE` = coreboot
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: Firmware](../../generic-test-setup/#firmware).
+
+**Test steps**
+
+1. Power on the DUT
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md:23]: Improved

File: docs/unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+1. While the DUT is booting, hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md:24]: Improved

File: docs/unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+1. Select the `UEFI Shell` option using the arrow keys and press `Enter`
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md:25]: Improved

**Comment by macpijan** [comment]
Merged to master via CLI

**Pull request closed without merging**